README.md

```
@@ -1,24 +1,15 @@
+
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/jordanpartridge/maestro.svg?style=flat-square)](https://packagist.org/packages/jordanpartridge/maestro)
+[![GitHub Tests Action Status](https://img.shields.io/github/actions/workflow/status/jordanpartridge/maestro/run-tests.yml?branch=main&label=tests&style=flat-square)](https://github.com/jordanpartridge/maestro/actions?query=workflow%3Arun-tests+branch%3Amain)
+[![GitHub Code Style Action Status](https://img.shields.io/github/actions/workflow/status/jordanpartridge/maestro/fix-php-code-style-issues.yml?branch=main&label=code%20style&style=flat-square)](https://github.com/jordanpartridge/maestro/actions?query=workflow%3A"Fix+PHP+code+style+issues"+branch%3Amain)
+[![Total Downloads](https://img.shields.io/packagist/dt/jordanpartridge/maestro.svg?style=flat-square)](https://packagist.org/packages/jordanpartridge/maestro)
+
 This is where your description should go. Limit it to a paragraph or two. Consider adding a small example.
 
 ## Support us
 
-[<img src="https://github-ads.s3.eu-central-1.amazonaws.com/:package_name.jpg?t=1" width="419px" />](https://spatie.be/github-ad-click/:package_name)
+[<img src="https://github-ads.s3.eu-central-1.amazonaws.com/maestro.jpg?t=1" width="419px" />](https://spatie.be/github-ad-click/maestro)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)